ui/dashboards/src/components/EditDashboardButton/EditDashboardButton.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Button } from '@mui/material';
+import PencilIcon from 'mdi-material-ui/PencilOutline';
+
+export interface EditDashboardButtonProps {
```

Comment:
Also notable that I am NOT adding a storybook for this component or exporting it from `index.ts` because this is intended as an internal implementation component, not as a library component for general use.

Reply:
Maybe just have it extend the MUIButton so you can override props where default don't make sense? (eg. like size="small")

Reply:
Since we're not exposing this one for external use, I'm inclined to keep it simple to start until we have a need to support other use cases.